src/_P118_Itho.ino

```diff
@@ -56,13 +56,15 @@
 // See https://gathering.tweakers.net/forum/list_messages/1690945 for more information
 // code/idea was inspired by first release of code from 'Thinkpad'
 
-#ifdef USES_P118
+#include "_Plugin_Helper.h"
+
+#ifdef USES_P118_disabled
```

Comment:
This can be reverted/re-enabled, I think?

Reply:
Only if it doesn't cause a bootloop...
Still have to look at it.

Reply:
I think you already fixed it by moving the `ICACHE_RAM_ATTR` decoration to the right place.

Reply:
Nope, that change wasn't the fix.